R/DataDiscoveryRetrieval.R

```diff
@@ -1,7 +1,7 @@
-#' Generate TADA-Compatible Dataset with WQP Data
+#' Generate TADA-Compatible Data with WQP Data
```

Comment:
This is minutiae, but Generate TADA-Compatible _dataframe_ would be more consistent with the other replacements in this commit. Also consider _from_ or _using_ WQP Data, since that makes it clearer the WQP data is an input.

Reply:
Agree & added a commit for this change